docs/src/intro.md

```diff
@@ -20,7 +20,7 @@ julia> s = SpaceShip(Person(:julia, 2009), [0.0, 0.0, 0.0], [0.0, 0.0, 0.0])
 SpaceShip(Person(:julia, 2009), [0.0, 0.0, 0.0], [0.0, 0.0, 0.0])
 ```
 Lets update the captains name:
-```jldoctest spaceship
+```jldoctest spaceship; filter = r" .*$"
```

Comment:
Not related to this PR, but I fixed this doctest for Julia 1.2 just by ignoring the detail in error message.